### PR TITLE
Cleaning up word wrap and consolidating images.

### PR DESCRIPTION
**Pull request by mirons-google**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,38 +58,56 @@ You don't have to worry about explicitly declaring the page's category.
...
+to update all three locations:
 
 - `_data/doc_categories.yml`: add/update category name or location
 - `_config.yml`: add/update category default for given directory
 - `_docs/...`: add/update directory name
 
 ### Customizing the Bootstrap Material Design Theme
 
-The material design theme for bootstrap is adapted from [FezVrasta/bootstrap-material-design](https://github.com/FezVrasta/bootstrap-material-design) (see it's [LICENSE in third_party/](https://github.com/GoogleCloudPlatform/forseti-security/blob/gh-pages/third_party/bootstrap-material-design/LICENSE.md)).
+The material design theme for bootstrap is adapted from
```
Comment (carise) [inline README.md]: Should this be "Material Design" and "Bootstrap", respectively?

File: README.md [hunk 2]
```diff
@@ -50,38 +58,56 @@ You don't have to worry about explicitly declaring the page's category.
...
+[FezVrasta/bootstrap-material-design](https://github.com/FezVrasta/bootstrap-material-design)
+(see it's
```
Comment (carise) [inline README.md]: see its

**Comment by mirons-google** [comment]
done, thanks!

**Comment by blueandgold** [comment]
Adding `cla: yes` label, as the PR author is a googler at the time of this PR was created.

**Pull request merged**